lib/puppetserver/ca/utils/parse.rb

```diff
@@ -0,0 +1,82 @@
+module Puppetserver
+  module Ca
+    module Utils
+      module Parse
```

Comment:
How do we feel about this name change? I figured ``Puppetserver::Ca::Utils::Utils`` didn't make much sense.

Reply:
I would prefer something like `Cli` maybe.

Reply:
I wouldn't go with just `Cli` since there's a class named that in the higher level `Puppetserver::Ca` module. But maybe `CliParsing` or something?